qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -14,7 +14,7 @@
         target_process = random\w*.exe
         rng_data_rex = "0x\w"
         driver_name = "viorng"
-        read_rng_cmd = for /l %i in (1, 1, 1000) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
```

Comment:
why only run once instead of 1000 times ?

Reply:
The aim of the command is just to verify rng basic function works well.
No need to run so many times, it takes longer time and egd performance is not good, run 1000 times will make this cmd always timeout.

Reply:
make sense